packages/components/src/GuidanceBlock/GuidanceBlock.tsx

```diff
@@ -193,55 +166,43 @@ export const GuidanceBlock = ({
             </>
           )}
         </div>
-        {actions?.primary && (
-          <Media query="(max-width: 767px)">
-            {(isMobile: boolean): JSX.Element => (
-              <div
-                className={classNames({
-                  noRightMargin: isMobile || componentIsMobile,
-                  rightMargin: !(isMobile || componentIsMobile) && layout === 'default',
-                })}
-              >
-                <div
-                  className={classNames(
-                    styles.buttonContainer,
-                    actions?.secondary && styles.secondaryAction,
-                  )}
-                >
-                  <WithTooltip tooltipProps={actions.primary.tooltip}>
-                    <Button
-                      icon={
-                        withActionButtonArrow ? (
-                          <Icon name="arrow_forward" isPresentational shouldMirrorInRTL />
-                        ) : undefined
-                      }
-                      iconPosition="end"
-                      {...actions.primary}
-                      fullWidth={isMobile || componentIsMobile}
-                    />
+        {actions?.primary || actionsSlot ? (
```

Comment:
Is this correct if the component has actions.primary and actionsSlot you will use the old deprecated actions instead?